organization/components/timeslot/TimeslotForm.tsx

```
@@ -0,0 +1,163 @@
+  const [formData, setFormData] = useState({
+    startDate: '',
+    endDate: '',
+    slotDuration: 60,
+  });
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFormData(prev => ({
+      ...prev,
+      [name]: name === 'slotDuration' ? parseInt(value) || 0 : value,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)